### PR TITLE
Add first draft of break project to revisit Hotel

### PR DESCRIPTION
**Pull request by droberts-sea**
Questions:
- Is this too much content
- Are the instructions for what they should produce and turn in clear?
- Am I missing anything in the comprehension prompts?
- Are the directions on the refactoring activity clear enough, or is this too open-ended?

### CODE REVIEW

File: hotel-revisited.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+- What classes does each implementation include? Are the lists the same?
+- Write down a sentence to describe each class.
+- How do the classes relate to each other? It might be helpful to draw a diagram on a whiteboard or piece of paper.
+- What **data** does each class store? Are they different for the two implementations?
+- What **methods** does each class have? Are they different for the two implementations?
+- Consider the `Order#total_price` method. In each implementation:
+    - Is logic to compute the price delegated to "lower level" classes like `ShoppingCart` and `CartEntry`, or is it retained in `Order`?
+    - Does `total_price` directly manipulate the instance variables of other classes?
+- Which implementation better adheres to the single responsibility principle?
```
Comment (CheezItMan) [inline hotel-revisited.md]: Maybe a bonus question on which implementation is more loosely coupled and why.

File: hotel-revisited.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+Now that we've got you thinking about design, spend some time to revisit the code you wrote for the Hotel project. For each class in your program, ask yourself the following questions:
+- What is this class's responsibility?
+    - You should be able to describe it in a single sentence.
+- Is this class responsible for exactly one thing?
+- Does this class take on any responsibility that should be delegated to "lower level" classes?
+- Is there code in other classes that directly manipulates this class's instance variables?
+
+### Activity
+
+Identify one place in your Hotel project where you believe your design could be improved. Describe in `design-activity.md` what changes you would need to make to actualize that improvement, and why.
```
Comment (CheezItMan) [inline hotel-revisited.md]: Maybe some explicit prompting to look for areas where a class takes on multiple roles, or directly modifies the attributes of another class.

File: hotel-revisited.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+#### Prompts
+
+Once you have read through the above code, add a file to your Hotel project called `design-activity.md`. This file will be submitted with the rest of this exercise. In that file, please respond to the following prompts:
+
+- What classes does each implementation include? Are the lists the same?
+- Write down a sentence to describe each class.
+- How do the classes relate to each other? It might be helpful to draw a diagram on a whiteboard or piece of paper.
+- What **data** does each class store? Are they different for the two implementations?
+- What **methods** does each class have? Are they different for the two implementations?
```
Comment (CheezItMan) [inline hotel-revisited.md]: Instead of, "Are they different," use, "How do they differ."

**Review by CheezItMan — COMMENTED** [review]
Answers:
- Is this too much content
  - It's a lot, more than Rails Zoo?  Yes, but not an enormous amount more, if they did CSS design work in Zoo.  If we were doing alouette, I'd say it's too much.  However since they're deciding what needs to be modified that lets them scale the problem.  That's both a plus and a minus.  
- Are the instructions for what they should produce and turn in clear?
  - I would add some instructions clarifying criteria on what to change, rather than just change something.  Since we did release the reference implementation, some may just imitate that.  
- Am I missing anything in the comprehension prompts?
  - I made a few suggestions, sorry I hit submit early.  
  - I would add maybe modification a user wants made and which implementation would be easier to modify and why.  Maybe some items are cheaper if bought in bulk.
- Are the directions on the refactoring activity clear enough, or is this too open-ended?
  - It's probably as clear as it can be, this is an open-ended design activity, they have to choose how to modify their project and make new design decisions.  I added a suggestion for things to look for when deciding what to change in-line. 

A few suggestions:

-  I'd include a link to some reading on the solid responsibility principal, just in case they're not so clear on it.  Or refer to the section on the Metz book.
-  I might suggest taking out the `CartEntry` class and in implementation A and just have Cart trying to handle it all as instance variables, messy I know, but very close to some student implementations of Hotel.

**Review by CheezItMan — APPROVED** [review]

**Comment by droberts-sea** [comment]
Closing b/c we don't actually use `standalone-exercises` anymore

**Pull request closed without merging**